src/components/query-search-list/query-search-list.js

```diff
@@ -0,0 +1,119 @@
+import {
+  useEffect,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classnames from 'classnames/bind';
+
+import { Button } from '@folio/stripes/components';
+
+import ScrollView from '../scroll-view';
+import {
+  FIRST_PAGE,
+  PAGE_SIZE,
+} from '../../constants';
+
+import styles from './query-search-list.css';
+
+const cx = classnames.bind(styles);
+
+const QuerySearchList = ({
+  collection,
+  fetch,
+  fullWidth,
+  itemHeight,
+  notFoundMessage,
+  renderItem,
+  scrollable,
+  type,
+}) => {
+  const [page, setPage] = useState(FIRST_PAGE);
+
+  useEffect(() => {
+    fetch(page);
+  }, [fetch, page]);
+
+  const {
+    totalLength,
+    items,
+    hasFailed,
+    errors,
+  } = collection;
+  const length = items.length;
+
+  if (hasFailed && !length) {
+    return (
+      <div className={styles.error}>
+        {errors[0].title}
+      </div>
+    );
+  }
+
+  if (!length) {
+    return notFoundMessage;
+  }
+
+  const getLoadMoreButton = () => {
+    const isUnloadedPagesPresent = (totalLength !== length) && !(length % PAGE_SIZE);
+
+    if (isUnloadedPagesPresent) {
+      return (
+        <div className={styles['button-wrapper']}>
+          <Button
+            style={{ width: '75%', margin: '1rem auto' }}
```

Comment:
Maybe add styles for this button in css?

Reply:
I tried but class doesn't not present in layout. The same button in `MultiColumnList` component use style instead of class too.   Probably this is a issue of `Button` component.

Reply:
Ok, in this case it's fine to keep like this